### DISCUSSION TITLE
blog/add-a-comment-section-to-your-next-js-blog

### DISCUSSION THREAD

**Discussion by giscus** [ibizabroker/tarungowda.com#10] [Announcements]
# blog/add-a-comment-section-to-your-next-js-blog

Implement an ad free, no tracking, no database, no subscription comment section in your nextjs blog which supports multiple languages using GitHub discussions via Giscus.

https://www.tarungowda.com/blog/add-a-comment-section-to-your-next-js-blog

**Comment by ibizabroker** [comment]
#### You can also write in markdown.
```tsx
export default function TestSnippet() {
  return (
    <div>
      <p> giscus supports code snippets too! </p>
    </div>
  )
}
```
